scripts/ci-build.bat

```diff
@@ -0,0 +1,2 @@
+call npm install
```

Comment:
Is this the file Bacon looks for? Is so, can we add a comment to the top of this explaining why this file is needed to ensure nobody thinks it is a dead file?

Reply:
Yes, this is the file that'll be run on jenkins/bacon. Will update the file with comment.